service/start.go

```diff
@@ -280,10 +280,47 @@ func (svc *service) launchLNBackend(ctx context.Context, encryptionKey string) e
 		LNDMacaroonHex, _ := svc.cfg.Get("LNDMacaroonHex", encryptionKey)
 		lnClient, err = lnd.NewLNDService(ctx, svc.eventPublisher, LNDAddress, LNDCertHex, LNDMacaroonHex)
 	case config.LDKBackendType:
-		Mnemonic, _ := svc.cfg.Get("Mnemonic", encryptionKey)
-		LDKWorkdir := path.Join(svc.cfg.GetEnv().Workdir, "ldk")
+		mnemonic, _ := svc.cfg.Get("Mnemonic", encryptionKey)
+		ldkWorkdir := path.Join(svc.cfg.GetEnv().Workdir, "ldk")
+
+		nodeLastStartTime, _ := svc.cfg.Get("NodeLastStartTime", "")
+
+		// for brand new nodes, consider enabling VSS
+		if nodeLastStartTime == "" && svc.cfg.GetEnv().LDKVssUrl != "" {
+			albyUserIdentifier, err := svc.albyOAuthSvc.GetUserIdentifier()
+			if err != nil {
+				logger.Logger.WithError(err).Error("Failed to fetch alby user identifier")
+				return err
+			}
+			if albyUserIdentifier != "" {
+				me, err := svc.albyOAuthSvc.GetMe(ctx)
+				if err != nil {
+					logger.Logger.WithError(err).Error("Failed to fetch alby user")
+					return err
+				}
+				// only activate VSS for Alby paid subscribers
+				if me.Subscription.Buzz {
```

Comment:
can we extract this check somewhere to keep the start function small here. 
and potentially also easier to run it independently of Alby (if somebody wants to self-host a VSS?)